src/main/kotlin/no/nav/helse/spade/behandlinger/KafkaBehandlingerRepository.kt

```diff
@@ -54,4 +55,31 @@ class KafkaBehandlingerRepository(stream: BehandlingerStream) {
       log.error("unknown error while fetching state store", err)
       Either.Left(Feilårsak.UkjentFeil)
    }
+
+   fun getBehandlingerForPeriode(fom: String, tom: String): Either<Feilårsak, List<JsonNode>> = try {
+      stateStore.all().asSequence().filter { keyval ->
+         keyval.value.any {
+            it.has("avklarteVerdier") && it.path("avklarteVerdier").has("medlemsskap")
+               && it.path("avklarteVerdier").path("medlemsskap").has("vurderingstidspunkt")
+               && LocalDate.parse(it.path("avklarteVerdier").path("medlemsskap").get("vurderingstidspunkt").textValue().substring(0, 10)) >= (LocalDate.parse(fom))
+               && LocalDate.parse(it.path("avklarteVerdier").path("medlemsskap").get("vurderingstidspunkt").textValue().substring(0, 10)) <= (LocalDate.parse(tom))
+         }
+      }.let {
+         it
```

Comment:
Mangler bare `be`, nå 😄
Jeg tror ikke `let { it }` gjør noe som helst?

Reply:
Det har du helt rett i 😅